src/activities/categorization/resource/board/category_flowers.qml

```diff
@@ -0,0 +1,116 @@
+/* GCompris
+ *
+ * Copyright (C) 2017 Aman Kumar Gupta <gupta2140@gmail.com>
+ *
+ * Authors:
+ *   Aman Kumar Gupta <gupta2140@gmail.com>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.6
+
+QtObject {
+    property bool isEmbedded: false
+    property bool allowExpertMode: true
+    property string imagesPrefix: "qrc:/gcompris/data/words/flowers/"
+    property var levels: [
+        {
+            "name": qsTr("Flowers"),
+            "image": imagesPrefix + "rose.jpg",
```

Comment:
can we have a flowers image instead to represent the category instead of just a rose?

Reply:
yes, there's a "bunch of flowers" image in the image dataset...we can use that one.